apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -35,6 +47,7 @@ struct LiveMessage {
     var chatItem: ChatItem
     var typedMsg: String
     var sentMsg: String
+    var sent: Bool
```

Comment:
```suggestion
    var sentMsg: String?
```

this is redundant, as you could make sentMsg optional